activerecord/test/cases/enum_test.rb

```diff
@@ -788,6 +788,18 @@ def self.name; "Book"; end
     assert_not_predicate book, :suffix?
   end
 
+  test "allow array syntax for enum backed by string column" do
+    klass = Class.new(ActiveRecord::Base) do
+      self.table_name = "books"
+      enum cover: [:hard, :soft]
```

Comment:
The Book model in `activerecord/test/models/book.rb` still uses the hash syntax. Should I update it?